janusgraph-core/src/main/java/org/janusgraph/graphdb/relations/AbstractTypedRelation.java

```diff
@@ -14,15 +14,19 @@
 
 package org.janusgraph.graphdb.relations;
 
-import org.janusgraph.core.*;
+import org.apache.tinkerpop.gremlin.structure.Direction;
+import org.apache.tinkerpop.gremlin.structure.Property;
+import org.apache.tinkerpop.gremlin.structure.Vertex;
+import org.apache.tinkerpop.gremlin.util.iterator.IteratorUtils;
+import org.janusgraph.core.InvalidElementException;
+import org.janusgraph.core.PropertyKey;
+import org.janusgraph.core.RelationType;
```

Comment:
I don't know if it's documented anywhere, but all the files I've seen group the JanusGraph imports at the top and then group TinkerPop and other imports below. I think it makes to group the TinkerPop imports down below where they were initially.

Reply:
@chupman It might be a remnant of the original rebranding from `com.thinkaurelius` (which were above `org` imports) to `org.janusgraph`. It would probably be nice to eventually do a pass through and organize imports across the project. In the meantime though I agree it's best to match existing file/project convention.